File: Praktikum 2/Python/PRAK108-2310817210004-Allano Lintang Ertantora.py
cyc = 5
#putaran
dst = 14
#jarak
cir = dst/cyc
#keliling taman
print("Diketahui :")
print("Pak Dengklek mengelilingi taman =", cyc, "Putaran")
print("Jarak tempuh Pak Dengklek =", dst, "Kilometer")
for i in range(2):
    print(" ")
print("Jawaban :")
print("Jari-jari taman yang dikelilingi Pak Denglek adalah", round((cir)/(2*3.14), 2), "Kilometer")

File: Praktikum 2/C/PRAK108-2310817210004-Allano-Lintang-Ertantora.c
#include <stdio.h>

int main() {
    float cyc = 5;
    //Putaran
    float dst = 14;
    //jarak
    float cir = dst / cyc;
    //keliling taman
    printf("Diketahui :\n");
    printf("Pak Dengklek mengelilingi taman = %.f Putaran\n", cyc);
    printf("Jarak tempuh Pak Dengklek = %.f Kilometer\n\n\n", dst);
    printf("Jawaban :\n");
    printf("Jari-jari taman yang dikelilingi Pak Denglek adalah %.2f Kilometer", (cir)/(2*3.14));
    
    return 0;
}